content/en/blog/_posts/2023-08-15-pv-last-phase-transtition-time.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: blog
+title: "Persistent Volume Last Phase Transition Time"
+date: 2023-09-20
+slug: persistent-volume-last-phase-transition-time
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+In Kubernetes v1.28, we have introduced a new alpha feature that aims to improve PersistentVolume (PV)
```

Comment:
In Kubernetes v1.28 -> In the recent Kubernetes v1.28 release

Reply:
Changed.